readme.md

```diff
@@ -61,6 +61,15 @@ Default: `Infinity`
 
 Milliseconds to wait before stopping the search and resolving the promise to `undefined`.
 
+##### expectEntireElement
+
+Type: `boolean`<br>
+Default: `true`
+
+Explicitly wait for the entire element to be loaded instead of just its opening HMTL tag. If this is set to `false`, a `elementReady('nav')` promise could resolve before the last menu item has been downloaded.
+
+Note: This is unrelated to the loading of images, videos, scripts, etc. It only follows the loading of the current document.
```

Comment:
Maybe we should include a use-case for setting this to `false`?

Something like this:

> Set this to `false` if you don't care about the children elements. For example, you only need to ensure the top level element exists or you just want to read attributes from it.

Reply:
I tried rewriting the whole paragraph. I think it's useful to remind the user that this is due to progressive/incremental HTML parsing before trying to explain what this option does.

> Since the current document’s HTML is downloaded and parsed gradually, elements may appear in the DOM before _all_ of their children are "ready."
> 
> By default, `element-ready` guarantees the element and all of its children have been parsed. This is useful if you want to interact with them or if you want to `.append()` something inside.
> 
> By setting this to `false`, `element-ready` will resolve the promise as soon as it finds the requested selector, regardless of its content. This is ok if you're just checking if the element exists or if you want to read/change its attributes.

Reply:
I also added it to the JSDoc